src/github.com/matrix-org/rageshake/submit.go

```diff
@@ -117,15 +128,46 @@ func (s *submitServer) saveReport(ctx context.Context, p payload) error {
 
 	if s.ghClient == nil {
 		// we're done here
+		log.Println("GH issue submission disabled")
 		return nil
 	}
 
 	// submit a github issue
-	owner := "richvdh"
-	repo := "test"
-	title := "Automated bug report"
+
+	ghProj := s.GithubProjectMappings[p.AppName]
+	if ghProj == "" {
+		log.Println("Not creating GH issue for unknown app", p.AppName)
+		return nil
+	}
+	splits := strings.SplitN(ghProj, "/", 2)
+	if len(splits) < 2 {
+		log.Println("Can't create GH issue for invalid repo", ghProj)
+	}
+	owner, repo := splits[0], splits[1]
+
+	var title string
+	if userText == "" {
+		title = "Untitled report"
+	} else {
+		// set the title to the first line of the user's report
+		if i := strings.IndexAny(userText, "\r\n"); i < 0 {
```

Comment:
Ohhhh, `IndexAny` takes a list of *characters*. That's disappointing. Never mind, carry on then.